index.html

```diff
@@ -1676,15 +1697,19 @@ <h2 id="AudioContext">
               <li>Set a <code>control thread state</code> to
               <code>suspended</code> on the <a>AudioContext</a>.
               </li>
-              <li>Send a <a>control message</a> to start processing.
+              <li>Let <dfn>pendingResumePromises</dfn> be an empty ordered list
```

Comment:
Why is this list needed? We don't need such a list for any of the other promises returned from suspend/resume/close.